docs/content/documentation/getting-started/overview.md

```diff
@@ -0,0 +1,202 @@
+<h1 class="title">
+  This is my blog made with Zola.
+</h1>
+{% endblock content %}
+```  
+
+This tells Zola that `index.html` extends our `base.html` file and replaces the block called "content" with the text between the `{% block content %}` and `{% endblock content %}` tags.
+
+### Content Directory
+
+Now let's add some content. We'll start by making a `blog` subdirectory in the `content` directory and creating an `_index.md` file inside it. This file tells Zola that `blog` is a [section](@/documentation/content/section.md), which is how content is categorized in Zola.
+
+```bash
+├── content
+│   └── blog
+│       └── _index.md
+```
+
+In the `_index.md` file, we'll set the following variables in [TOML](https://github.com/toml-lang/toml) format:
+
+```md
++++
+title = "List of blog posts"
+sort_by = "date"
+template = "blog.html"
+page_template = "blog-page.html"
++++
+```
+
+> Note that although no variables are mandatory, the opening and closing `+++` are required.
+
+*sort_by = "date"* tells Zola to use the date to order our section pages (more on pages below). *template = "blog.html"* tells Zola to use `blog.html` in the `templates` directory as the template for listing the Markdown files in this section. *page_template = "blog-page.html"* tells Zola to use `blog-page.html` in the `templates` directory as the template for individual Markdown files. For a full list of section variables, please see the [section](@/documentation/content/section.md) documentation. We will use *title = "List of blog posts"* in a template (see below).
+
+### Templates
+
+Let's now create some templates. In the `templates` directory, create a `blog.html` file with the following contents:
+
+```html
+{% extends "base.html" %}
+
+{% block content %}
+<h1 class="title">
+  {{ section.title }}
+</h1>
+<ul>
+  {% for page in section.pages %}
+  <li><a href="{{ page.permalink }}">{{ page.title }}</a></li>
+  {% endfor %}
+</ul>
+{% endblock content %}
+```
+
+As done by `index.html`, `blog.html` extends `base.html`, but this time we want to list the blog posts. The *title* we set in the `_index.md` file above is available to us as `{{ section.title }}`. In the list below the title, we loop through all the pages in our section (`blog` directory) and output the page title and URL using `{{ page.title }}` and `{{ page.permalink }}`, respectively. If you go to [127.0.0.1:1111/blog/](http://127.0.0.1:1111/blog/), you will see the section page for `blog`. The list is empty because we don't have any blog posts. Let's fix that now.
+
+### Markdown Content
+
+In the `blog` directory, create a file called `first.md` with the following contents:
+
+```md
++++
+title = "My first post"
+date = 2019-11-27
++++
+
+This is my first blog post.
+```
+
+The *title* and *date* will be avaiable to us in the `blog-page.html` template as `{{ page.title }}` and `{{ page.date }}`, respectively. All text below the closing `+++` will be available to us as `{{ page.content }}`.
```

Comment:
The text below will be available as HTML

Reply:
"All text below the closing `+++` will be available as HTML in the  `{{ page.content }}` variable."? I'm currently using ` ` for directories and template variables names (TOML variables are in italics). Should I change to bold for template variables?